js/core/utils/shadow-dom.js

```diff
@@ -0,0 +1,59 @@
+let constructedStyleSheet = null;
+
+function createConstructedStyleSheet(rootNode) {
+    try {
+        // eslint-disable-next-line no-undef
+        return new CSSStyleSheet();
```

Comment:
seems like we can write it this way to get rid of lint disable:
```suggestion
        return new window.CSSStyleSheet();
```

Reply:
ESLint does not allow to use `window` here